sources/packages/backend/apps/api/src/main.ts

```diff
@@ -24,6 +25,24 @@ async function bootstrap() {
   // Setting global prefix
   app.setGlobalPrefix("api");
 
+  // Using helmet
+  app.use(
+    helmet({
+      strictTransportSecurity: {
+        maxAge: 31536000,
+        includeSubDomains: true,
+        preload: true,
+      },
+    }),
+  );
+
+  // Adding headers not covered by helmet
+  app.use((_, res, next) => {
+    res.setHeader("Cache-Control", "no-store");
+    res.setHeader("Pragma", "no-cache");
```

Comment:
The`Pragma` seems deprecated: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Pragma
Besides that, it seems that it can be replaced by `Cache-Control: no-cache`.
Are these headers added based on some best practice or was it based on some Wava recommendation?

Reply:
Yes, "Pragma" header was used for HTTP/1.0 and is deprecated and shouldn't there be browsers old enough to use it. Wava scan still recommends it for compatibility:
![image](https://github.com/bcgov/SIMS/assets/78114138/be2b0745-2cf2-4a52-a154-d6ff48f036a3)
Mozilla [mentions ](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Pragma)that it is ok to be used for compatibility if we have "Cache-Control" header.
![image](https://github.com/bcgov/SIMS/assets/78114138/e3f0a1eb-3204-41ad-9604-730203fa52bd)

Reply:
For Wava, we do not need to follow the recommendation if we have a justification.
The recommendation from Mozilla is that we do not need to use it if we use `Cache-control: no-cache`, right?
I would recommend just using the `Cache-control: no-cache` which apparently would supersede the `no-store`. Please let me know we I am missing something.